authors/apps/authentication/tests/test_model.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework.test import APITestCase
```

Comment:
Model test are good but lets focus on testing enpoints. Please organize tests in folders relating to what your building e.g `auth`